### PR TITLE
modified error message with invalid ports

### PR DESCRIPTION
**Pull request by LuitelSamikshya**
This PR modifies the error and warning message with invalid ports.
Test:

```console
echo google.com | ./httpx -p 8000-7000 -v
echo google.com | ./httpx -p 66000 -v
echo https://google.com:777777777 | ./httpx -v
```

### CODE REVIEW

File: common/customports/customport.go [hunk 1]
```diff
@@ -28,6 +28,39 @@ func (c *CustomPorts) String() string {
 	return "Custom Ports"
 }
 
+// ValidateCustomPorts to validate the custom port range
+func ValidateCustomPorts(value string) {
```
Comment (Mzack9999) [inline common/customports/customport.go:32]: I think we should change the signature to `func ValidateCustomPorts(value string) error` and just return the errors here.

File: runner/options.go [hunk 2]
```diff
@@ -510,7 +511,15 @@ func (options *Options) ValidateOptions() error {
 	if len(options.OutputMatchCdn) > 0 || len(options.OutputFilterCdn) > 0 {
 		options.OutputCDN = true
 	}
-
+	// Validate the custom port range
+	if len(options.CustomPorts) > 0 {
+		for _, value := range options.CustomPorts {
+			potentialPorts := sliceutil.Dedupe(strings.Split(value, ","))
+			for _, port := range potentialPorts {
+				customport.ValidateCustomPorts(port)
```
Comment (Mzack9999) [inline runner/options.go:519]: here we should check if `customport.ValidateCustomPorts(xx)` returned an error and either propagate/print

**Review by Mzack9999 — CHANGES_REQUESTED** [review]

File: common/customports/customport.go (hunk 1 above)
Comment (LuitelSamikshya) [reply common/customports/customport.go]: Done.

**Review by Mzack9999 — APPROVED** [review]
```console
$ echo scanme.sh | go run . -p 8000-7000 -silent
[FTL] First value of port range should be lower than the last port from your range: [8000, 7000].
```

**Pull request merged**